pkg/applicationserver/io/web/webhooks.go

```diff
@@ -194,6 +201,10 @@ func (w *webhooks) handleDown(
 			webhandlers.Error(res, req, errWebhookNotFound.New())
 			return
 		}
+		if hook.Paused {
+			logger.Debug("Webhook is paused")
+			return
+		}
```

Comment:
We should return an HTTP 202 Accepted here. By default the handler would set 200 which is technically incorrect.

Reply:
Done.